astropy/io/votable/exceptions.py

```diff
@@ -1082,6 +1082,19 @@ class W54(VOTableSpecWarning):
     default_args = ('1.3',)
 
 
+class W55(VOTableSpecWarning):
+    """
+    When non-ASCII characters are detected when reading
+    a TABLEDATA value for a FIELD with datatype="char", we
```

Comment:
Nitpick: API doc formatting.
```suggestion
    a TABLEDATA value for a FIELD with ``datatype="char"``, we
```